src/config.ts

```diff
@@ -454,6 +453,10 @@ export const ASSET_SWAPPER_MARKET_ORDERS_V1_OPTS: Partial<SwapQuoteRequestOpts>
     sampleDistributionBase: 1.05,
     feeSchedule: FEE_SCHEDULE_V1,
     gasSchedule: GAS_SCHEDULE_V1,
+    exchangeProxyOverhead: (sourceFlags: number) =>
+        [SOURCE_FLAGS.Uniswap_V2, SOURCE_FLAGS.SushiSwap].includes(sourceFlags)
+            ? new BigNumber(3e4)
```

Comment:
Won't this get effectively added to the current fee schedule? So right now univ2 is set to 100k base, which is roughly the total tx cost of a uni VIP swap. This would make the optimizer see a pure uni route as 130k? Am I thinking about this right?